src/EditorFeatures/TestUtilities/MoveStaticMembers/TestMoveStaticMembersService.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.MoveStaticMembers;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.MoveStaticMembers
+{
+    internal class TestMoveStaticMembersService : IMoveStaticMembersOptionsService
+    {
+        private readonly string _destinationType;
+
+        private readonly ImmutableArray<string> _selectedMembers;
+
+        private readonly string _filename;
+
+        public TestMoveStaticMembersService(string destinationType, string fileName, ImmutableArray<string> members)
+        {
+            _destinationType = destinationType;
+            _filename = fileName;
+            _selectedMembers = members;
+        }
+
+        public MoveStaticMembersOptions GetMoveMembersToTypeOptions(Document document, INamedTypeSymbol selectedType, ISymbol? selectedNodeSymbol)
+        {
+            var selectedMembers = selectedType.GetMembers().WhereAsArray(symbol => _selectedMembers.Contains(symbol.Name));
+            // just return all the selected members
+            return new MoveStaticMembersOptions(
+                _filename,
+                string.Join(".", selectedType.ContainingNamespace.ToDisplayString(), _destinationType),
```

Comment:
You're doing `string.Join`, then splitting it back in the constructor. Why not pass them separately?

Reply:
My idea was that the user could include a additional nested namespace(s) when they defined the type. For example, if they were moving a member from a class called `SomeNamespace.InnerNamespace.Class1`, and wanted to move it to a separate utility namespace, they could write `Utilities.Class1Helpers` in the destination selection box, and the new created class would be fully qualified as `SomeNamespace.InnerNamespace.Utilities.Class1Helpers`. 

I think splitting the extra namespaces from the destination box (`Utilities` in the example) and adding them on to the containing namespace, then passing in both separately would work as well.